commands/core/config.drush.inc

```diff
@@ -462,7 +462,7 @@ function _drush_config_export($destination, $destination_dir, $branch) {
     }
     $comment .= "\n\n$output";
 
-    if (!$commit && !drush_confirm(dt('The .yml files in your export directory (!target) will be deleted.', array('!target' => $destination_dir)))) {
+    if (!$commit && !drush_confirm(dt('The .yml files in your export directory (!target) will be deleted and overwridden with the active config.', array('!target' => $destination_dir)))) {
```

Comment:
Perhaps we dont need this prompt. Its already not shown when --commit is passed. It should always be safe to re-export. Thoughts?